### ISSUE TITLE
Discord Channel

### ISSUE THREAD

**Issue by ArnyminerZ** [leikoilja/ha-google-home#277]
Now that this project is getting bigger and bigger I think the discussions panel is hard to follow. It's a bit unordered, and the conversations are a bit strange.

I'd suggest creating a Discord channel, or other platforms such as Slack for discussing new features, and implementing new code, also joining both the `glocaltokens` package and this one, so the discussions about each other can be related.

How do you see it? Any other suggestions? @KapJI @leikoilja

**Comment by leikoilja** [comment]
I agree that we could use a bit more structure in communication. I have never tried using Discord, mostly Slack. I would upvote Slack, just to avoid installing another tool :D 
@ArnyminerZ if you feel for creating a new workspace - that would be awesome, then we can all join it :)

**Comment by ArnyminerZ** [comment]
Here's the link for joining the Slack Workgroup

https://join.slack.com/t/newworkspace-hen3594/shared_invite/zt-s8qez63p-wWGJde2gGf2oJgLU48IHJw

**Comment by ArnyminerZ** [comment]
Added the Slack link at #280

**Comment by ArnyminerZ** [comment]
Please, @leikoilja install the Slack integration for Github 😅 

https://slack.github.com/github/v2/install/10655107/eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ0cmlnZ2VyX2lkIjoiMjIxMzM0MDQ3OTU0MS4yMjEzMjA4MDYxNzMzLjk2MmNjZDAwZWVmMmVmMzVkYWEwOGE4OTYxN2ExNWZlIiwidGVhbUlkIjoiVDAyNjk2NDFUTUsiLCJjbGllbnREb21haW4iOiJoYS1nb29nbGUtaG9tZSIsInVzZXJJZCI6IlUwMjVYRVlRWlRSIiwiY2hhbm5lbElkIjoiRDAyNjVIWTQ5NzAiLCJ0cmlnZ2VyaW5nQXBwIjoiQTAxQlA3UjRLTlkiLCJyZXBsYXlDb21tYW5kIjp0cnVlLCJpYXQiOjE2MjQ4Njg5MTAsImV4cCI6MTYyNDg3MjUxMH0.aWt76vIfxWExk54XVRZyq1RmFpD1MvJ8LAUkWd7PONA

**Comment by leikoilja** [comment]
Great! Have added Slack to this repo 💥 
@ArnyminerZ can you please make some of us admins in the slack workspace as well? :)

**Comment by leikoilja** [comment]
@KapJI, @DurgNomis-drol, @Drakulix feel free to join us as well :)

**Comment by KapJI** [comment]
I don't use neither Slack nor Discord and I'd prefer to keep important discussions here on Github.

**Comment by DurgNomis-drol** [comment]
I am with @KapJI.

**Comment by leikoilja** [comment]
haha, the excitement was short, @ArnyminerZ, hhaha :D But i also get it, it makes sense to skip this step for now :) Thanks for your initiative thou
